src/main/java/com/fabriccommunity/spookytime/enchantment/BeheadingEnchantment.java

```diff
@@ -3,6 +3,7 @@
 import com.fabriccommunity.spookytime.registry.SpookyEnchantments;
 import com.fabriccommunity.spookytime.registry.SpookyItems;
 import com.google.common.collect.ImmutableList;
+import net.minecraft.block.AnvilBlock;
```

Comment:
Is this used somewhere?

Reply:
Oh, yeah, no. Still not used to IDEA's import organization so I missed it.